formlibrary/models/session.py

```diff
@@ -0,0 +1,16 @@
+from django.db import models
+from .case import Individual
```

Comment:
Please add an empty line over here.
PEP8: Imports should be grouped in the following order:

Standard library imports.
Related third party imports.
Local application/library specific imports.

Reply:
@epanjwani This one is still pending.